solvers/heatTransferFoamI/ReadFiles/readSystemProperties.H

```diff
@@ -60,3 +82,5 @@ dimensionedScalar temporal3
 (
     systemProperties.lookup("temporal3")
 );
+
+double mechanicalTest = readScalar(systemProperties.lookup("mechanicalTest"));
```

Comment:
```suggestion
Switch mechanicalTest(systemProperties.lookup("mechanicalTest"));
```
or `readBool`, but never `double`.